src/pages/mine.tsx

```diff
@@ -51,11 +51,12 @@ const Mine: FC = () => {
                 <p>Something went wrong! Please contact admin</p>
               ) : (
                 <>
-                  {tasks.length > 0 ? (
-                    <div>{CardList(tasks)}</div>
-                  ) : (
-                    <p>No Tasks Found</p>
-                  )}
+                  {
+                    tasks.length > 0 && <div>{CardList(tasks)}</div>
+                  }
+                  {
+                    response?.length == 0 && tasks.length == 0 && <p>No Tasks Found</p>
```

Comment:
I checked the code, this response object holds the tasks array right? so, if yes then why are we even checking the state (task state), why not a simple check of `response.length > 0 ? {render tasks} : no tasks`, do check if you can do this.